src/catacomb.rs

```diff
@@ -29,8 +30,11 @@ use crate::drawing::{Graphics, MAX_DAMAGE_AGE};
 use crate::input::TouchState;
 use crate::orientation::{Accelerometer, AccelerometerSource};
 use crate::output::Output;
-use crate::shell;
 use crate::window::Windows;
+use crate::{daemon, shell};
+
+/// The script to run on compositor start.
+const ON_START_SCRIPT: &str = ".config/catacomb/on_start.sh";
```

Comment:
I think it's better to have this just the filename, if we're joining with home path anyway.

Also I'm not sure I'd call it "on_start", maybe "post_start" is better? Or "after_start" if you prefer to keep the language a little more basic.

Reply:
post start is fine, I guess.